web/src/components/pages/profile/avatar-setup/avatar-setup.css

```diff
@@ -26,18 +26,19 @@
 
     .clip-title {
         position: relative;
-        left: 20%;
-        right: 20%;
+        left: 15%;
+        right: 15%;
         padding-top: 10px;
     }
     .group-1 {
         width: 508px;
         height: 169px;
         object-fit: contain;
         background-image: url('./group-1.svg');
+        background-repeat: no-repeat;
         @media only screen and (max-width: 375px) {
```

Comment:
I don't think we need the media query at all here, see this example: https://jsfiddle.net/zk2wnLpj/
We just define a max-width and when you shrink the viewport, it takes up all the available widht (and breaks correctly)

Reply:
Ok . I am looking.